RELEASE_NOTES/2.12.0.md

```diff
@@ -0,0 +1,218 @@
+<!-- THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. -->
+
+# Fluid Framework v2.12.0
+
+## Contents
+
+- [Fluid Framework v2.12.0](#fluid-framework-v2120)
+  - [Contents](#contents)
+  - [✨ New Features](#-new-features)
+    - [New APIs to create and load containers without using the Loader object (#22902)](#new-apis-to-create-and-load-containers-without-using-the-loader-object-22902)
+      - [Overview](#overview)
+        - [Use `createDetachedContainer` to create a detached container](#use-createdetachedcontainer-to-create-a-detached-container)
+        - [Use `loadExistingContainer` to load an existing container](#use-loadexistingcontainer-to-load-an-existing-container)
+        - [Use `rehydrateDetachedContainer` to create a detached container from a serializedState of another container](#use-rehydratedetachedcontainer-to-create-a-detached-container-from-a-serializedstate-of-another-container)
+        - [Note on `ICreateAndLoadContainerProps`.](#note-on-icreateandloadcontainerprops)
+      - [Change details](#change-details)
+  - [⚠️ Deprecations](#️-deprecations)
+    - [SummarizerStopReason, ISummarizeEventProps, and ISummarizerEvents are now deprecated (#23217)](#summarizerstopreason-isummarizeeventprops-and-isummarizerevents-are-now-deprecated-23217)
+      - [Change details](#change-details-1)
+    - [IContainerRuntimeOptions.enableGroupedBatching is now deprecated (#23260)](#icontainerruntimeoptionsenablegroupedbatching-is-now-deprecated-23260)
+      - [Change details](#change-details-2)
+    - [IContainerRuntimeOptions.flushMode is now deprecated (#23288)](#icontainerruntimeoptionsflushmode-is-now-deprecated-23288)
+      - [Change details](#change-details-3)
+    - [Merge-Tree and SharedString ISegment Deprecations (#23323)](#merge-tree-and-sharedstring-isegment-deprecations-23323)
+      - [Change details](#change-details-4)
+    - [The ContainerRuntime class is now deprecated (#23331)](#the-containerruntime-class-is-now-deprecated-23331)
+      - [Change details](#change-details-5)
+    - [🛠️ Start Building Today!](#️-start-building-today)
+
+## ✨ New Features
+
+### New APIs to create and load containers without using the Loader object ([#22902](https://github.com/microsoft/FluidFramework/issues/22902))
+
+#### Overview
+
+Provide standalone APIs to create and load containers instead of using the Loader object to do so. Earlier hosts were supposed to create the Loader object first and then call methods on it to create and load containers. Now they can just utilize these APIs directly and get rid of the Loader object.
```

Comment:
Nit.
```suggestion
Provide standalone APIs to create and load containers instead of using the Loader object to do so. Before, hosts were supposed to create the Loader object first and then call methods on it to create and load containers. Now they can just utilize these APIs directly and get rid of the Loader object.
```